### ISSUE TITLE
NcColorPicker doesn't have proper accessibility labels

### ISSUE THREAD

**Issue by GVodyanov** [nextcloud/nextcloud-vue#8142]
Missing:
- `aria-selected="true"` for selected colors
- Semantic information such as name, role, and value is missing because no WAI-ARIA attributes are defined

**Comment by ShGKme** [comment]
> * `aria-selected="true"` for selected colors

Where exactly? `NcColorPicker`'s color options are native radio button `<input type="radio" />`. It has a native selected state.

**Comment by ShGKme** [comment]
> * Semantic information such as name, role, and value is missing because no WAI-ARIA attributes are defined

But it does have `aria-label`

<img width="1261" height="506" alt="Image" src="https://github.com/user-attachments/assets/03a3e5b8-f3e1-4161-b837-a389719303ee" />

**Comment by ShGKme** [comment]
I'll close the issue. If it was about a different place or a different problem, feel free to reopen
